### PR TITLE
Add page about contributing to this repo

### PR DESCRIPTION
**Pull request by addshore**
Bug: T272369

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+diagrams.net is used to create refined SVG diagrams.
+
+These SVGs should only be edited using a diagrams.net editor.
+
+If you are using VS Code, you can take advantage of the [diagrams.net integration plugin](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio).
+
+If you are not using VS Code you can edit diagrams on [diagrams.net iteself](https://www.diagrams.net/), exporting as an SVG.
+
+### [mermaid](https://mermaid-js.github.io/mermaid/#/)
```
Comment (Silvan-WMDE) [inline CONTRIBUTING.md:60]: Do we actually use mermaid diagrams at the moment? If not, maybe we can remove it to not confuse people about which one is needed for editing the diagrams?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Source
+
+Source for the documentation lives on [Github](https://github.com/wmde/wikidata-wikibase-architecture).
+
+While working on this source locally, you can install needed dependencies with `npm install`.
+
+Commands exist for other local actions:
+
+- `npm run test` - Runs basic markdown and SVG
```
Comment (Silvan-WMDE) [inline CONTRIBUTING.md:22]:
```suggestion
- `npm run test` - Runs basic markdown and SVG linting
```

File: docs/decisions/README.md [hunk 3]
```diff
@@ -1 +1,19 @@
-Decisions were made and recorded here
+# Decisions
+
+While creating this Architecture documentation some decisions were made.
+
+When those decisions to not seem to fit inside the system documentation the decisions will be recorded here.
```
Comment (Silvan-WMDE) [inline docs/decisions/README.md:5]:
```suggestion
When those decisions do not seem to fit inside the system documentation the decisions will be recorded here.
```

**Review by Silvan-WMDE — APPROVED** [review]
I like it, with a few suggestions.

File: CONTRIBUTING.md (hunk 1 above)
Comment (addshore) [reply CONTRIBUTING.md:60]: Not yet, but I'm 99% sure we will do in the coming days with the interaction diagrams in client

Comment (addshore) [reply CONTRIBUTING.md:60]: If we don't end up using it then we can remove it.

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**